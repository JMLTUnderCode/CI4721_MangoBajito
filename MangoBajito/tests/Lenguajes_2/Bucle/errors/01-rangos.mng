# Test 01: Error de rangos de iteracion.
se_prende(){
	culito n : mango = 0;
	repite_burda i entre 1 hasta 4.2 {
		n = n + 1;
	};
	repite_burda i entre -10 hasta 4.2e+207 {
		n = n + 1;
	};
	repite_burda i entre -10 hasta 's' {
		n = n + 1;
	};
	repite_burda i entre -10 hasta "20" {
		n = n + 1;
	};
	repite_burda i entre -10 hasta Sisa {
		n = n + 1;
	};
};
