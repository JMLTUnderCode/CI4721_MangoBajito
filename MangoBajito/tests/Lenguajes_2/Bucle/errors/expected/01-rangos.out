| AcomÃ³date que te lo estan analizando d:-)
Error: TYPE_ERROR
  - Sendo peo en la linea 4, columna 35: Tas en droga?. Tienes a los tipos entre:'mango' y hasta:'manguita' los dos deben ser 'mangos', loca perdia'.
  - Sendo peo en la linea 7, columna 42: Tas en droga?. Tienes a los tipos entre:'mango' y hasta:'manguangua' los dos deben ser 'mangos', loca perdia'.
  - Sendo peo en la linea 10, columna 37: Tas en droga?. Tienes a los tipos entre:'mango' y hasta:'negro' los dos deben ser 'mangos', loca perdia'.
  - Sendo peo en la linea 13, columna 38: Tas en droga?. Tienes a los tipos entre:'mango' y hasta:'higuerote' los dos deben ser 'mangos', loca perdia'.
  - Sendo peo en la linea 16, columna 38: Tas en droga?. Tienes a los tipos entre:'mango' y hasta:'tas_claro' los dos deben ser 'mangos', loca perdia'.
[1;31m[5m
               =======================================================
                       ---->        Error Program        <----        
               =======================================================
[0m
| Hay un poquito de cÃ³digo en tu marihuana d:-]
